styles.css

```diff
@@ -514,12 +876,53 @@ textarea {
 
 @media only screen and (min-width: 375px) {
   body {
-    font-size: 16px;
+    background-image: url('img/header-illsutration-mobile.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+  .container {
+    padding-top: 50px;
+  }
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 50px;
+    z-index: 1;
+  }
+  nav {
+    position: absolute;
+    top: 0;
+    left: -50px;
+    width: 100%;
+    z-index: 1;
   }
```

Comment:
- [ ] As mentioned previously please make sure you're mobile version is not getting disrupted by the changes you make to the desktop as they should stay consistent with the given design: 👇
- Here's how it looks currently:
  ![firefox_n7iIPS4gLm](https://github.com/NgachaIan/PersonalPortfolio/assets/104968236/fa50f7d7-73ae-4d9b-97fe-02f7142d5159)

Reply:
@AbelG101 how do I include both html files and their corresponding files and make sure they load differently in the same project?